cmd/soroban-cli/src/commands/contract/install.rs

```diff
@@ -125,7 +120,23 @@ pub(crate) fn build_install_contract_code_tx(
         cond: Preconditions::None,
         memo: Memo::None,
         operations: vec![op].try_into()?,
-        ext: TransactionExt::V0,
+        ext: TransactionExt::V1(SorobanTransactionData {
+            resources: SorobanResources {
+                footprint: LedgerFootprint {
+                    read_only: VecM::default(),
+                    read_write: vec![ContractCode(LedgerKeyContractCode { hash: hash.clone() })]
+                        .try_into()?,
+                },
+                // TODO: what values should be used here?
+                instructions: 0,
+                read_bytes: 0,
+                write_bytes: 0,
+                extended_meta_data_size_bytes: 0,
+            },
+            // TODO: what value to use here?
```

Comment:
I discussed that with @dmkozh and @sisuresh. Leave it like this for now, and once the simulateTransaction endpoint is ready, we'll need to update this code to use the endpoint to figure out these values.

Reply:
Yes, I was planning to do that :) 

However, the question is, what values to use for cases in which we were not preflight-ing  the transaction (e.g. when uploading the contract).

Will we now need to start preflight-ing  those transactions or can the values be inferred (like we used to do before introducing fees)